lib/Migrator.php

```diff
@@ -62,12 +62,8 @@ public function initialize()
      *
      * @return VersionInterface[] Executed migrations
      */
-    public function migrate($to = null, OutputInterface $output)
+    public function migrate(?string $to, OutputInterface $output)
```

Comment:
Did change this back in: https://github.com/phpcr/phpcr-migrations/pull/13/commits/12662952cb820dbace672fac8f6e2c0a11d11710

Reply:
I don't think so. The minimum version required is 7.2 even if it is not specified in composer but we can change it to phpdoc instead.

Reply:
Did not mean that its not compatible with PHP Version but public function should not add any typehints or other things in minor or hotfix releases, as somebody could call or extend from that class and php would then error which it would not before. So I would avoid adding any typehints aslong as we plan to release a new major version.

Reply:
Changed !